### ISSUE TITLE
default to UMAP view

### ISSUE THREAD

**Issue by paolo-kunderfranco** [aertslab/SCENIC#117]
Dear All,
I would like to set default view as UMAP instead of tSNE,

Where do I change this parameter?

Many Thanks
Paolo

**Comment by r-mvl** [comment]
I would also be interested in plotting in UMAP space instead of tSNE... would this be feasible somehow?

Many thanks

**Comment by BioInfoUCI** [comment]
Hello,

I have the same question. 

Thank you!

**Comment by paolo-kunderfranco** [comment]
still no response?

**Comment by Mevelo** [comment]
no... I just found how to do this outside the SCENIC workflow, just for AUCell. But it would be good if the authors could advise. Looking at the issues opened on the github page there doesn't seem to have been much support from the lab for a while :(

**Comment by paolo-kunderfranco** [comment]
exactly, I was able also to plot with AUC some specific regulons in UMAP.

I agree, the support misses..
